flatgfa/src/packedseq.rs

```diff
@@ -204,20 +285,46 @@ pub fn create_slice(vec: &PackedVec, s: std::ops::Range<usize>) -> PackedSlice<'
 
 /// Returns a vector containing the base pairs referenced by `slice`
 pub fn get_slice_seq(slice: PackedSlice<'_>) -> Vec<Nucleotide> {
-    slice.vec_ref.get_range(slice.span)
+    slice.vec_ref.as_ref().get_range(slice.span)
+}
+
+pub fn total_bytes(seq: &PackedSeqView) -> usize {
+    let num_elems = seq.len();
+    let seq_size = if num_elems % 2 == 1 {
+        (num_elems / 2) + 1
+    } else {
+        num_elems / 2
+    };
+
+    let toc_size = std::mem::size_of::<PackedToc>();
+    toc_size + seq_size
+}
+
+pub fn import(filename: &str) -> Vec<Nucleotide> {
```

Comment:
I touched on this above in `cmds.rs`, but I think we would want such a utility to return a `PackedSeqView`. The current signature requires a `Vec` allocation and a copy of all the data, which seems unnecessary to require for all clients.